source/Kepler-System-Tests/SystemInstallationTest.class.st

```diff
@@ -0,0 +1,56 @@
+"
+Unit test for SystemInstallation
+"
+Class {
+	#name : #SystemInstallationTest,
+	#superclass : #TestCase,
+	#category : #'Kepler-System-Tests'
+}
+
+{ #category : #tests }
+SystemInstallationTest >> testCreatedSystem [
+
+	| rootSystem |
+
+	rootSystem := (SampleSystemInstallation installing:{SampleProjectModule. SampleCustomerModule}) value: '1.0'.
```

Comment:
`value:` does not seem like a good name for the execution of a system installation